src/core/reducers/user.js

```diff
@@ -61,6 +61,10 @@ export const hasPermission = (
   return permissions.includes(permission);
 };
 
+export const isAdmin = (state: { user: UserStateType }): boolean => {
+  return hasPermission(state, ADMIN_SUPER_POWERS);
```

Comment:
What if this did...

```js
if (permissions.includes(ADMIN_SUPER_POWERS)) {
  return true;
}
```

...and then call `isAdmin(state)` from within `hasPermission()`? I guess you'd need a duplicate check on `if (!state.user.permissions)` so I'm not sure it's worth it.

Reply:
Yeah, I did not want to duplicate the check so I'd prefer keep this as is.